src/index.js

```diff
@@ -8,6 +8,20 @@ const supportMultiple = (typeof document !== 'undefined' && document && document
   true;
 
 class Dropzone extends React.Component {
+  static renderChildren(children, isDragActive, isDragReject) {
+    if (!children) {
+      return children;
+    }
+    let renderedChildren = children;
+    const props = { isDragActive, isDragReject };
```

Comment:
Let's rename `props` to something with more context. Right now it's easy to confuse with `this.props`